keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -46,7 +50,10 @@ class ApplicationController(
     val hasDeliveryConfig = deliveryConfigRepository.hasDeliveryConfig(application)
 
     if (includeDetails) {
-      val resources = resourceRepository.getSummaryByApplication(application)
+      var resources = resourceRepository.getSummaryByApplication(application)
+      if (resourcePauser.applicationIsPaused(application)) {
+        resources = resources.map { it.copy(status = PAUSED) }
+      }
```

Comment:
It would be really nice to include the paused status of the application in the details. Without that, we'll need either a) a separate endpoint to get the paused status of a single application, or b) call the endpoint to get a list of all paused applications (like we do now with `ApplicationVeto`s) and check if the application is in the list (which is kind of wasteful).

With this information, we'll be able to determine if a resource is explicitly paused, or whether it's paused because the application itself is paused. With that information, a user can either unpause a resource or unpause an application through the UI.

I would prefer that to overwriting the status of the resources in the response here.

Reply:
Makes sense, I'll make that change.

Reply:
I’m going back and forth on this. I don’t know which is approach is right! 

As long as the application pause state is included at the application level, we can figure it out in the UI. There's a valid argument to be made that showing the resources with whatever status they have is confusing to API consumers. But... I dunno.

Reply:
🤷‍♀ I think the only way we will figure this out is through using it. 

I think it's slightly more clear to show every resource as paused, but I'm hoping that users don't look at this endpoint and instead look at the UI so that it won't matter.

If you form a stronger opinion let me know and I'll make any changes needed!